core/src/main/java/org/apache/calcite/rel/metadata/RelMdColumnUniqueness.java

```diff
@@ -68,6 +79,15 @@ public class RelMdColumnUniqueness
       ReflectiveRelMetadataProvider.reflectiveSource(
           new RelMdColumnUniqueness(), BuiltInMetadata.ColumnUniqueness.Handler.class);
 
+  /**
+   * If a row has a unique column, then an aggregation that returns a value from that column is
+   * also unique. These are the aggregations that do that. Note that this quality is not
+   * guaranteed in the presence of an OVER clause. NOTE: if a multi-argument function is added,
+   * methods that use this Set must be enhanced to select the appropriate column to pass through.
+   */
+  static final Set<SqlKind> PASSTHROUGH_AGGREGATIONS =
```

Comment:
The first sentence should be a description of the field. "The set of aggregate functions A such that if x is unique then A(x) will also be unique. An aggregate function with this property is called 'passthrough'."